dev/Vagrantfile

```diff
@@ -172,7 +172,17 @@ end
 def prepare
   action = ARGV[0]
   if action =~ /^(provision|up|reload)$/
-    installer = prompt_installer
+    installer = if ENV.has_key? 'FORCE_INSTALLER'
+                  if ENV['FORCE_INSTALLER'] =~ /^.*chef-server-core.*deb$/ and File.file?(ENV['FORCE_INSTALLER'])
+                    puts "Valid $FORCE_INSTALLER override detected. Using #{ENV['FORCE_INSTALLER']} as Chef Server 12 build."
+                    ENV['FORCE_INSTALLER']
+                  else
+                    puts "$FORCE_INSTALLER set to #{ENV['FORCE_INSTALLER']} but is not a valid Chef Server 12 build."
+                    exit 1
+                  end
+                else
+                  prompt_installer
+                end
```

Comment:
`$INSTALLER` still requires human interaction, it just adds `$INSTALLER` to the list to select from. Since my scripts are spinning DVM up and down automatedly, it needs a way to bypass that menu in general.